### ISSUE TITLE
Puppeteer: Login sessions still remain even when fresh fixtures are loaded

### ISSUE THREAD

**Issue by grafitto** [huridocs/uwazidocs#5012]
**Describe the bug**
Some E2E tests fail with a timeout for not finding `a[aria-label="Sign in"]`

**To Reproduce**
1.Run 2 tests where one does not logout in `afterAll`, make sure the one without logout runs first

**Expected behavior**
If the second test requires login, chances are it will fail with a timeout for not finding `a[aria-label="Sign in"]`

This behavior is because the sessions are not stored in the same db, and therefore not cleared when fresh fixtures are loaded. Therefore there is already a user logged in even when fixtures reset.

**Potential fixes**
1. Always logout after tests
2. Have a global `afterAll` that logs out
3. Have a way to also reset sessions when fixtures are reset

**Comment by RafaPolit** [comment]
Could this be that you committed the fixtures with the login credentials? And you are actually loading the login credentials on each reset?

**Comment by LaszloKecskes** [comment]
Sessions are stored in uwazi_shared_db, which is not exported with the fixtures, I think. But I've updated the fixtures at one point for the pdfjs issue, and while I've tried to remove unrelated things before dumping, if you think that might be related, please double check it as Rafa suggests.

**Comment by txau** [comment]
@konzz I guess we can close this one since it is a puppeteer related issue?
